src/IssueViz.Security/HotspotsList/TableDataSource/CustomColumns/PriorityTableColumnDefinition.cs

```diff
@@ -29,13 +29,12 @@ namespace SonarLint.VisualStudio.IssueVisualization.Security.HotspotsList.TableD
     internal class PriorityTableColumnDefinition : TableColumnDefinitionBase
     {
         public const string ColumnName = "ReviewPriority";
+        public const int Width = 120;
 
         public override string Name { get; } = ColumnName;
 
-        public override double DefaultWidth => 50;
+        public override string DisplayName { get; } = "Priority";
 
-        public override string DisplayName => "Priority";
-
-        public override bool IsFilterable => true;
```

Comment:
IsFilterable=true is the default value, so no need to specify it.

Reply:
Perhaps not from a functionality point of view, but it doesn't help the reader unless they happen to know it is the default...

Reply:
There are other properties, such as IsSortable, IsResizable, IsCopyable - it's noisy to repeat them and just set to default values.